Exercises and solutions for chapter 10 (Model Interpretation) (#526)
mlr-org/mlr3book: book/interpretation.qmd

```

Blue dot stands for the prediction for Steve. Only a big change in bill length could convince the model of Steve's different species.

## Exercises

Model explanation allows us to confront our expert knowledge related to the problem with relations learned by the model. 
Following tasks are based on predictions of the value of football players based on data from the FIFA game. 
It is a graceful example, as most people have some intuition about how a footballer's age or skill can affect their value. 

> The latest FIFA statistics can be downloaded from [kaggle.com](https://www.kaggle.com/), but also one can use the 2020 data avaliable in the `DALEX` packages(see `DALEX::fifa` dataset). The following tasks can be performed in both the `iml` and `DALEX` packages so choose any of them to solve them.

Reviewer: ```suggestion
The latest FIFA statistics can be downloaded from [kaggle.com](https://www.kaggle.com/), but also one can use the 2020 data avaliable in the `DALEX` packages(see `DALEX::fifa` dataset).
The following exercises can be performed in both the `iml` and `DALEX` packages and we have provided solutions for both.
```
